### DISCUSSION TITLE
The issue view in projects is missing history and references

### DISCUSSION THREAD

**Discussion by nicksloan** [community/community#35093] [Projects and Issues]
In a project, the issue view is missing lots of history, including adding removing labels and milestones and mentions from other issues and pull requests. This gets in the way of using Projects effectively because I can't tell that (for example) there is an open PR for an issue without leaving the project to go to the issue page.

The project issue view should more closely mirror the actual issue page, and it should definitely show all of the information that is captured on the issue page.

**Comment by jd-santos** [comment]
Agreed. I've been stumped for a few months now because the documentation mentioned that child issues have a link to the parent, but I rarely open the full issue link. It's not just me, my entire team has been manually pasting links to parent issues because none of us were aware this was available.

**Comment by jbrown-xentity** [comment]
Our team has been frustrated by this as well. We implement epics as part of the board (as a separate tag), so we can only see epics when looking at the board, and we can only see mentioned issues/PR's when in the issue view itself (outside of the board). This is a really important feature for us!

**Comment by dk-morinibu** [comment]
There is another side to this issue. The Github Actions trigger by adding labels is fired by the history of the issue page, so labeling in the project issue view does not fire that trigger. Solving this problem will help automate Issue management!
